tests/settings.py

```diff
@@ -10,6 +10,8 @@
 
 SECRET_KEY = 'secret'
 
+RESOLWE_API_HOST = 'localhost'
```

Comment:
I think @tjanez has some other ideas to support this for tests. He'll write more about it.